docs/index.md

```diff
@@ -39,4 +39,4 @@ If you are struggling with Docker, [Docker Troubleshooting](./docker/docker-trou
 For the frequently asked questions, see [FAQ and Common Issues](./introduction/faq-and-common-issues.md).
 
 ## Documenting your own project
-We strongly believe that not only the framework itself needs documentation, but also your your project deserves it's own docs. The tips for writing project documentation are written down in [Guidelines for Project Documentation](./project/guidelines-for-project-documentation.md).
\ No newline at end of file
+We strongly believe that not only the framework itself needs documentation, but also your your project deserves it's own docs. The tips for writing project documentation are written down in [Guidelines for Project Documentation](./project/guidelines-for-project-documentation.md).
```

Comment:
I was trying really hard to figure out what has changed here :laughing:  now I see you removed the trailing spaces :+1: I noticed there is the word "your" twice though.

```suggestion
We strongly believe that not only the framework itself needs documentation, but also your project deserves it's own docs. The tips for writing project documentation are written down in [Guidelines for Project Documentation](./project/guidelines-for-project-documentation.md).
```